net/znc/files/znc.init

```diff
@@ -156,45 +155,30 @@ add_user() {
 	echo "</User>" >> $ZNC_CONFIG
 }
 
-
-start() {
+start_service() {
 	config_load znc
 	config_foreach znc_global znc
 
-	if [ "$DISABLED" -eq 1 ]; then
-		return 0
-	fi
+	[ "$DISABLED" -eq 0 ] || return 0
```

Comment:
I felt a non-zero value would indicate disabled.  In previous code if someone set the value to anything other than 1, then it mean enabled.